app/webhook.go

```diff
@@ -59,12 +59,15 @@ func runWebhookServer(c *cli.Context) error {
 		return errors.Wrap(err, "unable to get client config")
 	}
 
-	s := server.New(context.Background(), cfg, namespace)
+	done := make(chan struct{})
```

Comment:
NIT: Generally I prefer using a cancellable context for example, this way you can pass down the context.
And you can wait on the ctx.done channel :)